### PR TITLE
stat manager: Token Informations added in protobuf message.

### PR DESCRIPTION
**Pull request by Netmisa**
# Introduction

_Token informations added in protobuf message (token_id and token)_
## 1. Dependencies

Depends of:
- https://github.com/CanalTP/navitia-proto/pull/33
# Don't forget
## Rebuild navitia protobuf

``` javascript
protoc -Inavitia-proto --python_out=navitiacommon/ navitia-proto/*.proto
```

### CODE REVIEW

**Comment by lrocheWB** [comment]
Test failed: jormungandr_integration_tests ......***Failed

**Comment by pbougue** [comment]
Jenkins, test this please

**Comment by pbougue** [comment]
You can see details here: https://ci.navitia.io/job/navitia_pull_request_build/2080/testReport/
The problem seems to be the access to SHA-1 of `navitia-proto`.
I merge `proto`.
For the next PR we recommend pushing the `proto` branch to the main repo and open the PR from there (not from a fork) so that it can be tested by jenkins.

**Comment by pbougue** [comment]
Jenkins, test this please.

**Comment by pbougue** [comment]
You have to add `navitia-proto` SHA-1 to that PR

File: source/navitia-proto [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 21feade1da44dbb4e76f9c0b64106cbf3c239b55
+Subproject commit e0749c29e7f01f8921fcc92c7b3680de0fe66d55
```
Comment (TeXitoi) [inline source/navitia-proto:1]: Commit is OK (navitia-proto/master)

File: source/jormungandr/jormungandr/stat_manager.py [hunk 2]
```diff
@@ -37,6 +37,7 @@
 from jormungandr import app
 from jormungandr.authentication import get_user, get_token, get_app_name
 from jormungandr import utils
+from navitiacommon.models import Key
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/stat_manager.py]: Why?

Comment (kinnou02) [inline source/jormungandr/jormungandr/stat_manager.py]: Why import key?

Comment (Netmisa) [inline source/jormungandr/jormungandr/stat_manager.py]: Thanks I forgot to remove it

**Comment by lrocheWB** [comment]
Works for me :+1:

**Pull request merged**